script.js

```diff
@@ -1,4 +1,87 @@
+var gameMode = 'player1rolls';
+var player1Dice1;
+var player1Dice2;
+var player1Pick;
+var player2Dice1;
+var player2Dice2;
+var player2Pick;
+
+var diceRoll = function () {
+  // number from 0 to 1
+  var digit = Math.random();
+  // number from 0 to 6
+  var number = digit * 6;
+  // integer to float from 0 to 5
+  var integer = Math.floor(number) + 1;
+  return integer;
+};
+
+var player1turn = function () {
+  player1Dice1 = diceRoll();
+  player1Dice2 = diceRoll();
+  gameMode = 'player1picks';
+};
+
+var player2turn = function () {
+  player2Dice1 = diceRoll();
+  player2Dice2 = diceRoll();
+  gameMode = 'player2picks';
+};
+
+var player1P = function (input) {
+  var result;
+  if (input == 1) {
+    result = (player1Dice1 * 10) + player1Dice2;
+  }
+  else if (input == 2) {
+    result = (player1Dice2 * 10) + player1Dice1;
+  }
+  gameMode = 'player2rolls';
+  return result;
+};
+
+var player2P = function (input) {
```

Comment:
`player1P` and `player2P` use the same logic, any way to refactor the logic so we don't repeat ourselves? Repeating logic increases possibility of bugs and makes code harder to maintain.